lighthouse-core/audits/byte-efficiency/unminified-css.js

```diff
@@ -14,8 +14,21 @@ const UIStrings = {
   /** Imperative title of a Lighthouse audit that tells the user to minify (remove whitespace) the page's CSS code. This is displayed in a list of audit titles that Lighthouse generates. */
   title: 'Minify CSS',
   /** Description of a Lighthouse audit that tells the user *why* they should minify (remove whitespace) the page's CSS code. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
-  description: 'Minifying CSS files can reduce network payload sizes. ' +
-  '[Learn more](https://developers.google.com/web/tools/lighthouse/audits/minify-css).',
+  description: {
+    message: 'Minifying CSS files can reduce network payload sizes.' +
+      ' $link_start$Learn More$link_end$',
+    description: 'Description of a Lighthouse audit that tells the user *why* they should ' +
+      'minify (remove whitespace) the page\'s CSS code. This is displayed after a user expands ' +
+      'the section to see more. No character length limits.',
+    placeholders: {
+      link_start: {
+        content: '[',
+      },
+      link_end: {
+        content: '](https://developers.google.com/web/tools/lighthouse/audits/minify-css)',
```

Comment:
this is a pretty unfortunate syntax. Maybe we want to come up with something that expands to this in `collect-strings.js`?

Reply:
> Maybe we want to come up with something that expands to this in collect-strings.js?

Just to be clear, I meant this as a super low priority idea that we can think about while we do everything else first and implement it later if there's a nice fix. It kind of sucks to look at/edit but we could live with it if need be (we may need a test to make sure a typo in one of the contents doesn't accidentally break a link)

Reply:
That was also my thought.  This is super heavyweight to have in each `.js` file.  I can try to experiment.

Reply:
```
something that expands to this in collect-strings.js
```

Let's just keep it as `[markdown](format)` and expand it to additional placeholders in collect strings. Seems simple enough.

Reply:
Like auto convert `[` and `](url)` into `link_start` `link_end` in collect strings?

Reply:
yeah - I think that'd be dope